pages/api/cidades/v1/[estado].js

```diff
@@ -0,0 +1,33 @@
+import cidades from 'cidades-promise';
+import microCors from 'micro-cors';
+
+const CACHE_CONTROL_HEADER_VALUE = 'max-age=0, s-maxage=86400';
+const cors = microCors();
+
+/* retorna todas as cidades por meio da sigla do estado
+ * exemplo da rota: estado/v1/sp 
+ */
+async function Cidades(request, response) {
+    const requestedCidades = request.query.estado;
```

Comment:
Um padrão que foi combinado no início com o @lucianopf foi de todo o código ser escrito em inglês e o restante em português. Comentamos um pouco disso nessa issue: https://github.com/filipedeschamps/BrasilAPI/issues/8

Então tanto o nome do arquivo (e por consequência da propriedade recebida pela query `estado`) deveria ser `state`